services/src/main/java/org/keycloak/organization/authentication/authenticators/browser/OrganizationAuthenticator.java

```diff
@@ -91,6 +101,12 @@ public void action(AuthenticationFlowContext context) {
 
             if (broker.isEmpty()) {
                 // not a managed member, continue with the regular flow
+                if (organization != null) {
+                    context.form().setAttributeMapper(attributes -> {
```

Comment:
It might make sense to change the authentication flow a bit to not ask for the username/email again once the user is identified. See https://github.com/keycloak/keycloak/issues/30339.

By updating the context with the user (e.g.: `context.setUser`) we change the flow to only ask for credentials in the second step and avoid setting the `OrganizationBean` here.